### ISSUE TITLE
Installation fails due to conflicting redis version

### ISSUE THREAD

**Issue by NeolithEra** [leffss/ScrapyRedisBloomFilterBlockCluster#1]
Hi, users are unable to run **_ScrapyRedisBloomFilterBlockCluster_** due to dependency conflict with _**redis**_ package. 
As shown in the following full dependency graph of **_ScrapyRedisBloomFilterBlockCluster_**, **_ScrapyRedisBloomFilterBlockCluster_** requires _**redis>=2.10**_，while **_redis-py-cluster_** requires _**redis==2.10.6**_.

According to pip’s “first found wins” installation strategy, _**redis==3.3.6**_ is the actually installed version. However, _**redis==3.3.6**_ does not satisfy _**redis==2.10.6**_.

### Dependency tree
```
scrapy-redis-bloomfilter-block-cluster-1.0.1
| +-redis(version range:>=2.10)
| +-redis-py-cluster(version range:>=1.3.4)
| | +-redis(version range:==2.10.6)
| +-scrapy(version range:>=1.4)
| | +-queuelib(version range:*)
| | +-twisted(version range:>=13.1.0,<=19.2.0python)
| | +-twisted(version range:>=13.1.0python)
| | +-w3lib(version range:>=1.17.0)
| +-six(version range:>=1.5.2)
| +-twisted(version range:==18.9.0)
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Solution

1.	Fix your direct dependency to be redis==2.10.6 redis-py-cluster==1.3.6 . 
I have checked this revision will not affect your downstream projects now.
2.	Fix your direct dependency to be redis-py-cluster==1.3.4，and redis>=2.10.2
3.	Ask your upstream project redis-py-cluster to loose the version range of redis.

Which solution do you prefer, 1 or 2?
@leffss Please let me know your choice. I can submit a PR to solve this issue.

**Comment by leffss** [comment]
Repaired in lastest version.
